english/advice_from_mother/advice_from_mother.md

```diff
@@ -7,6 +7,6 @@ into the bar.
 
 just a random test
 
-My mother also told me that I should eat my vegetables.
+My mother also told me that I should eat my vegetables. Something about them being good for me. But at that time, I was at the age that I thought "If it doesn't taste good, it just can't be good for me".
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.